app/views/posts/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Home#index</h1>
+
+<% @posts.each do |post| %> 
```

Comment:
Posts need to be displayed in a timeline manner, so `@posts` should be sorted / ordered by date and time, preferably most recently created post first.

[`REQUIRED`]

Reply:
Hi @adriaanbd, thanks for the review

I have currently this scope in my post model (app/ models/post.rb)

`  default_scope { order(created_at: :desc) }`

Wouldn't that be working as such?

Reply:
Oh @samgaco yes you are right, somehow I missed this. Since no dates/time is displayed on the UI I wasn't able to confirm it. You should have gotten a pass, let me tell Alvaro to correct this. Sorry about this!

Reply:
No problem. I will work on those test moving on, thanks a lot for your review :+1: . It was very helpful, I definitely need to improve my testing skills.